packages/terra-site/src/examples/date-picker/DatePickerIncludeDates.jsx

```diff
@@ -2,11 +2,29 @@ import React from 'react';
 import moment from 'moment';
 import DatePicker from 'terra-date-picker';
 
-const DatePickerExample = () => (
-  <DatePicker
-    name="date-input-include"
-    includeDates={[moment().format(), moment().subtract(1, 'days').format(), moment().add(1, 'days').format()]}
-  />
-);
+class DatePickerExample extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = { date: '' };
+    this.handleDateChange = this.handleDateChange.bind(this);
+  }
+
+  handleDateChange(event, date) {
+    this.setState({ date });
+  }
+
+  render() {
+    return (
+      <div>
+        <p>Selected ISO Date: {this.state.date}</p>
+        <DatePicker
+          name="date-input-include"
+          includeDates={[moment().format(), moment().subtract(1, 'days').format(), moment().add(1, 'days').format()]}
+          onChange={this.handleDateChange}
+        />
+      </div>
+    );
+  }
+}
```

Comment:
All of these DatePicker examples share an identical state management system. They could potentially be wrapped with a generic state driven component to ensure consistency and less code duplication / future changes